src/orders/components/OrderCustomerAddressesEditDialog/utils.ts

```diff
@@ -33,3 +50,58 @@ export function validateDefaultAddress<
   }
   return defaultAddress;
 }
+
+export const getAddressEditProps = (
+  variant: "shipping" | "billing",
+  data: OrderCustomerAddressesEditData,
+  handlers: OrderCustomerAddressesEditHandlers,
+  change: FormChange,
+  dialogErrors: Array<OrderErrorFragment | AccountErrorFragment>,
+  setAddressSearchState: (
+    newState: React.SetStateAction<OrderCustomerSearchAddressState>
+  ) => void,
```

Comment:
If we're using react types, let's be consistent 

```suggestion
  setAddressSearchState: React.Dispatch<React.SetStateAction<OrderCustomerSearchAddressState>>;
```